lib/elixir/lib/supervisor.ex

```diff
@@ -529,12 +528,12 @@ defmodule Supervisor do
           | {:max_seconds, pos_integer}
 
   @typedoc "Supported strategies"
-  @type strategy :: :one_for_one | :one_for_all | :rest_for_one
+  @type strategy :: :one_for_one | :one_for_all | :rest_for_one | :simple_one_for_one
```

Comment:
Eventhough it has been deprecated, it should still be reflected in the typespecs

Reply:
We have recently agreed in another issue that we won'tkeep in typespecs as we do want the warning there still.